### PR TITLE
feat: add pytorch lightning snippet to cookbook

### PR DESCRIPTION
**Pull request by Kelton8Z**

### CODE REVIEW

File: .github/2.0/cookbooks/Executor.md [hunk 1]
```diff
@@ -506,3 +506,28 @@ Line number corresponds to the 1.x code:
...
+
+import torch
+
+class plMwuAutoEncoder(Executor):
+     def __init__(self, **kwargs):
+        super().__init__()
+
+     @requests
+     def encode(self, docs, **kwargs):
+         ae = AE(input_height=32).from_pretrained('cifar10-resnet18')
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:527]: this should go in the `init`. U do not want to start it everytime

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: .github/2.0/cookbooks/Executor.md [hunk 2]
```diff
@@ -506,3 +506,27 @@ Line number corresponds to the 1.x code:
...
+     def __init__(self, **kwargs):
+        super().__init__()
+        self.ae = AE(input_height=32).from_pretrained('cifar10-resnet18')
+     
+     @requests
+     def encode(self, docs, **kwargs):
+         for doc in docs:
+             doc.embedding = self.ae(torch.from_numpy(doc.blob)).detach().numpy()
+
+         ae.freeze()
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:531]: this does not work now, This would crash

Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:531]: The freeze should go into the `init` not after encoding.

Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:531]: The freeze should go into the `init` not after encoding.

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: .github/2.0/cookbooks/Executor.md (hunk 2 above)
Comment (Kelton8Z) [reply .github/2.0/cookbooks/Executor.md:531]: my bad

File: .github/2.0/cookbooks/Executor.md [hunk 3]
```diff
@@ -534,6 +534,31 @@ Line number corresponds to the 1.x code:
...
+        self.ae.freeze()
+         
+     @requests
+     def encode(self, docs, **kwargs):
+         for doc in docs:
+             input_tensor = torch.from_numpy(doc.blob)
+             output_tensor = self.ae(input_tensor)
+             doc.embedding = output_tensor.detach().numpy()
+```
+=======
```
Comment (JoanFM) [inline .github/2.0/cookbooks/Executor.md:561]: Remove

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**